Rewrite/src/Rewrite.CSharp/CSharpPrinter.cs

```diff
@@ -584,9 +636,14 @@ public override J VisitBlock(J.Block block, PrintOutputCapture<TState> p)
             {
                 p.Append("=>");
                 VisitStatements(block.Padding.Statements, JRightPadded.Location.BLOCK_STATEMENT, p);
+                if (block.Statements.FirstOrDefault() is not Cs.ExpressionStatement) // expression statements print their own semicolon
+                {
+                    p.Append(";");
```

Comment:
Again: Is the trivia before semicolon covered?

Reply:
Seems to work fine as it's covered by unit tests